internal/config/config.go

```diff
@@ -402,10 +404,8 @@ func readConfig(configDir string) (*viper.Viper, error) {
 	cfgViper.SetEnvPrefix(DefaultEnvPrefix)
 	cfgViper.SetConfigName(DefaultConfigName)
 	cfgViper.SetConfigType(DefaultConfigType)
-	cfgViper.AddConfigPath(configDir)              // adding provided directory as search path
-	cfgViper.AddConfigPath(DefaultConfigDirectory) // adding home directory as search path
```

Comment:
should we search these other paths still? If someone passes in a dir that doesn't exist / is empty, and has a default config would it be an error? Maybe an unexpected success?

Reply:
The problem is that we need a deterministic place to write the file.__